roles/pulp_common/tasks/repos.yml

```diff
@@ -85,11 +85,12 @@
 
 # Corresponds to the rhel8+ CodeReady Linux Builder repo
 # Foreman reminds you to wear a yellow hardhat when using power tools.
+# On our box PowerTool repo is lower case
```

Comment:
You no longer need these 3 changes because of https://github.com/pulp/pulp_installer/pull/507 . In fact, since your rebase, all your change does is break CentOS 8.2 compatibility, your box is CentOS 8.3.

Reply:
changes remove, and rebased